schemas/2019-05-01/Microsoft.NetApp.json

```diff
@@ -0,0 +1,671 @@
+{
+  "id": "https://schema.management.azure.com/schemas/2019-05-01/Microsoft.NetApp.json#",
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "Microsoft.NetApp",
+  "description": "Microsoft NetApp Resource Types",
+  "resourceDefinitions": {
+    "netAppAccounts": {
```

Comment:
>netAppAccounts [](start = 5, length = 14)

please make changes in the root templates:
2014-04-01-preview/deploymentTemplate.json, 
2015-01-01/deploymentTemplate.json,
2019-04-01/deploymentTemplate.json.

Reply:
> 2014-04-01-preview/deploymentTemplate.json,
> 2015-01-01/deploymentTemplate.json,
> 2019-04-01/deploymentTemplate.json.

Ok, entries added.